src/MatrixClient.ts

```diff
@@ -915,6 +915,27 @@ export class MatrixClient extends EventEmitter {
             .then(response => response["content_uri"]);
     }
 
+    /**
+     * Download content from the homeserver's media repository.
+     * @param {string} mxcUrl the mxcUrl URL.
+     * @param {string} allowRemote Indicates to the server that it should not attempt to fetch the media if it is deemed remote. This is to prevent routing loops where the server contacts itself. Defaults to true if not provided.
```

Comment:
```suggestion
     * @param {string} allowRemote Indicates to the server that it should not attempt to fetch the 
     * media if it is deemed remote. This is to prevent routing loops where the server contacts itself. 
     * Defaults to true if not provided.
```